.travis.yml

```diff
@@ -10,8 +10,6 @@ jobs:
   include:
     - env: OPAMWITHTEST=false
```

Comment:
why do we bother with `OPAMWITHTEST=false`?

Reply:
I don't think it makes sense either, I've updated the PR